addons/account/models/account_partial_reconcile.py

```diff
@@ -483,9 +483,10 @@ def _create_tax_cash_basis_moves(self):
                 partial = partial_values['partial']
 
                 # Init the journal entry.
+                move_date = partial.max_date if partial.max_date > (move.company_id.period_lock_date or date.min) else fields.Date.context_today(self)
```

Comment:
I would just store `today = fields.Date.context_today(self)` out of the loops

Reply:
Updated, thanks